src/Views/BucketPanel/EntryDetail.tsx

```diff
@@ -181,6 +285,153 @@ export default function EntryDetail(props: Readonly<Props>) {
           onRemoved={() => history.push(`/buckets/${bucketName}`)}
         />
       )}
+
+      <Modal
+        title="Upload File"
+        open={isUploadModalVisible}
+        onCancel={() => {
+          setIsUploadModalVisible(false);
+          setUploadError("");
+          uploadForm.resetFields();
+          setUploadFile(null);
+          setLabels([]);
+        }}
+        footer={null}
+        width={600}
+      >
+        <Form
+          form={uploadForm}
+          onFinish={handleUpload}
+          layout="vertical"
+          style={{ padding: "20px 0" }}
+        >
+          <Form.Item style={{ marginBottom: "24px" }}>
+            <Upload.Dragger
+              beforeUpload={(file) => {
+                // Add file size check (e.g., 10MB limit)
+                const maxSize = 10 * 1024 * 1024; // 10MB
+                if (file.size > maxSize) {
+                  setUploadError(
+                    `File size must be smaller than ${maxSize / (1024 * 1024)}MB`,
+                  );
+                  return false;
+                }
+                setUploadError("");
+                setUploadFile(file);
+                return false;
+              }}
+              maxCount={1}
+              style={{ padding: "24px" }}
+            >
+              <p className="ant-upload-drag-icon" style={{ fontSize: "32px" }}>
+                <UploadOutlined />
+              </p>
+              <p
+                className="ant-upload-text"
+                style={{ fontSize: "16px", margin: "16px 0 8px" }}
+              >
+                Click or drag file to this area to upload
+              </p>
+              <p className="ant-upload-hint" style={{ color: "#666" }}>
+                Maximum file size: 10MB
+              </p>
+            </Upload.Dragger>
+          </Form.Item>
+
+          <Form.Item
+            label={<Typography.Text strong>Timestamp</Typography.Text>}
+            name="timestamp"
+            style={{ marginBottom: "24px" }}
+          >
+            <DatePicker
+              showTime
+              style={{ width: "100%" }}
+              placeholder="Select date and time (optional)"
+            />
+          </Form.Item>
+
```

Comment:
I think the data type is still needed, because sometimes we need to override types. So:
1) If the field has MIME type  check the format and use it if it's valid
2) If the filed empty:
  * try to detect from the file
  * if we can't keep it empty , the storage assigns the default one